server/dash/dashboards/neighborhood_recent.py

```diff
@@ -174,19 +181,58 @@ def update_table(selected_council):
     Input("council_list", "value")
 )
 def update_text(selected_council):
+    """Dash Callback Function to update the indicator cards at the top of the dashboard.
```

Comment:
ditto, please change to descriptive style

Reply:
Thanks for the suggestion - resolved.